tests/benchmarks/mbedtls/testcase.yaml

```diff
@@ -3,4 +3,6 @@ common:
   tags: crypto
 tests:
   benchmark.crypto.mbedtls:
-    platform_whitelist: qemu_x86 frdm_k64f sam_e70_xplained nrf52840dk_nrf52840
+    filter: CONFIG_TEST_RANDOM_GENERATOR
+    integration:
+      - qemu_x86
```

Comment:
Is there a reason `frdm_k64f sam_e70_xplained nrf52840dk_nrf52840` are removed from the integration testing list?

I can see the value of running these "integration tests" on them, since `mbedtls` is a crypto library and should ideally be tested/validated on every major supported platform if not all.

Reply:
> I can see the value of running these "integration tests" on them, since `mbedtls` is a crypto library and should ideally be tested/validated on every major supported platform if not all.

validated how? We are just building those tests right now, integration testing limits that to only 1 runnable platform in CI. Removing the whitelisting here enables this on every single platform we have including those that were whitelisted and those that were not.

Reply:
> validated how? We are just building those tests right now, integration testing limits that to only 1 runnable platform in CI. Removing the whitelisting here enables this on every single platform we have including those that were whitelisted and those that were not.

@nashif I was under the impression that the `integration:` platform list was migrated with the future support for hardware CI testing in mind (i.e. I see other commits in this PR that left `frdm_k64f` and `sam_e70_xplained` in the `integration:` list).

Reply:
If I understood correctly from the discussion about the `integration` list it supposes to contain only emulated platforms where tests/samples will be run by zephyr's CI. I've seen `fedm_k64f` and `sam_e70_xplained ` added to the integration list in cmsis_dsp tests but (again if I got it correct from @nashif description) it was due to a 'hassle' with being able to add the `integration` list and all the samples/tests will have to be still reevaluated to correctly handle the `integration`.

Reply:
That is not clear for me. What is the criteria used to pick where to do the integration test ?

Related to this specific test,it should be tested in different platforms, mbedtls has some integration with platform that should be tested as well, for example random number generation. This benchmarking test is exercising several functions and would be nice to have them running in as much targets as possible.